### DISCUSSION TITLE
Feature request: Support for a header view

### DISCUSSION THREAD

**Discussion by animaonline** [software-mansion/react-native-screens#981] [Ideas]
Pretty much what the title says.

**Comment by tboba** [comment]
Hi @animaonline, sorry for the late response 🙏 
I don't quite understand what do you mean by supporting for a header views 😄 Right now you can control what should be placed in a header with three props that are supported on both Android and iOS: `headerLeft`, `headerCenter` and `headerRight`. 
Is this also what you wanted? If not, could you describe what do you mean by thinking about header views?

**Reply by animaonline** [reply]
Hi @tboba.

Thanks for getting back to me, tbh I don't remember what the original issue was 😅

However, I think I was looking for something like headerCenter, maybe it wasn't available at the time of my comment?

**Reply by tboba** [reply]
You're probably right, from what I can see those props could be added 2 years ago, which coincides the date this discussion has been created 😄 
Let me close this discussion then, but in case you recall what you've meant let us know! 

Cheers 🤞
